# Copyright (C) 2021 Aleksandr Migunov

# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 3 of the License, or
# (at your option) any later version.

# This program is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
# GNU General Public License for more details.
  
# You should have received a copy of the GNU General Public License
# along with this program; if not, write to the Free Software
# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.

from PySide import QtCore, QtGui

class Ui_MainWindow(object):
    def setupUi(self, MainWindow):
        MainWindow.setObjectName("MainWindow")
        MainWindow.resize(800, 600)
        self.centralwidget = QtGui.QWidget(MainWindow)
        self.centralwidget.setObjectName("centralwidget")
        self.gridLayout = QtGui.QGridLayout(self.centralwidget)
        self.gridLayout.setObjectName("gridLayout")
        self.label_Text_1 = QtGui.QLabel(self.centralwidget)
        self.label_Text_1.setObjectName("label_Text_1")
        self.gridLayout.addWidget(self.label_Text_1, 0, 0, 1, 1)
        self.label_Text_2 = QtGui.QLabel(self.centralwidget)
        self.label_Text_2.setObjectName("label_Text_2")
        self.gridLayout.addWidget(self.label_Text_2, 0, 1, 1, 1)
        self.textEdit_Text_1 = QtGui.QTextEdit(self.centralwidget)
        self.textEdit_Text_1.setObjectName("textEdit_Text_1")
        self.gridLayout.addWidget(self.textEdit_Text_1, 2, 0, 1, 1)
        self.textEdit_Text_2 = QtGui.QTextEdit(self.centralwidget)
        self.textEdit_Text_2.setObjectName("textEdit_Text_2")
        self.gridLayout.addWidget(self.textEdit_Text_2, 2, 1, 1, 1)
        self.textEdit_Text_3 = QtGui.QTextEdit(self.centralwidget)
        self.textEdit_Text_3.setObjectName("textEdit_Text_3")
        self.gridLayout.addWidget(self.textEdit_Text_3, 2, 2, 1, 1)
        self.label_Text_3 = QtGui.QLabel(self.centralwidget)
        self.label_Text_3.setObjectName("label_Text_3")
        self.gridLayout.addWidget(self.label_Text_3, 0, 2, 1, 1)
        MainWindow.setCentralWidget(self.centralwidget)
        self.menubar = QtGui.QMenuBar(MainWindow)
        self.menubar.setGeometry(QtCore.QRect(0, 0, 800, 25))
        self.menubar.setObjectName("menubar")
        self.menuFile = QtGui.QMenu(self.menubar)
        self.menuFile.setObjectName("menuFile")
        self.menuNew = QtGui.QMenu(self.menuFile)
        self.menuNew.setObjectName("menuNew")
        self.menuOpen = QtGui.QMenu(self.menuFile)
        self.menuOpen.setObjectName("menuOpen")
        self.menuSave = QtGui.QMenu(self.menuFile)
        self.menuSave.setObjectName("menuSave")
        self.menuEdit = QtGui.QMenu(self.menubar)
        self.menuEdit.setObjectName("menuEdit")
        self.menuUndo = QtGui.QMenu(self.menuEdit)
        self.menuUndo.setObjectName("menuUndo")
        self.menuRedo = QtGui.QMenu(self.menuEdit)
        self.menuRedo.setObjectName("menuRedo")
        self.menuCut = QtGui.QMenu(self.menuEdit)
        self.menuCut.setObjectName("menuCut")
        self.menuCopy = QtGui.QMenu(self.menuEdit)
        self.menuCopy.setObjectName("menuCopy")
        self.menuPaste = QtGui.QMenu(self.menuEdit)
        self.menuPaste.setObjectName("menuPaste")
        self.menuFind = QtGui.QMenu(self.menuEdit)
        self.menuFind.setObjectName("menuFind")
        self.menuSelect_All = QtGui.QMenu(self.menuEdit)
        self.menuSelect_All.setObjectName("menuSelect_All")
        self.menuTools = QtGui.QMenu(self.menubar)
        self.menuTools.setObjectName("menuTools")
        self.menuFont = QtGui.QMenu(self.menuTools)
        self.menuFont.setObjectName("menuFont")
        self.menuColor = QtGui.QMenu(self.menuTools)
        self.menuColor.setObjectName("menuColor")
        self.menuHighlight = QtGui.QMenu(self.menuTools)
        self.menuHighlight.setObjectName("menuHighlight")
        self.menuView = QtGui.QMenu(self.menubar)
        self.menuView.setObjectName("menuView")
        self.menuZoom_In = QtGui.QMenu(self.menuView)
        self.menuZoom_In.setObjectName("menuZoom_In")
        self.menuZoom_Out = QtGui.QMenu(self.menuView)
        self.menuZoom_Out.setObjectName("menuZoom_Out")
        self.menuHelp = QtGui.QMenu(self.menubar)
        self.menuHelp.setObjectName("menuHelp")
        MainWindow.setMenuBar(self.menubar)
        self.statusbar = QtGui.QStatusBar(MainWindow)
        self.statusbar.setObjectName("statusbar")
        MainWindow.setStatusBar(self.statusbar)
        self.actionExit = QtGui.QAction(MainWindow)
        self.actionExit.setObjectName("actionExit")
        self.actionNew_Text_1 = QtGui.QAction(MainWindow)
        self.actionNew_Text_1.setObjectName("actionNew_Text_1")
        self.actionNew_Text_2 = QtGui.QAction(MainWindow)
        self.actionNew_Text_2.setObjectName("actionNew_Text_2")
        self.actionOpen_Text_1 = QtGui.QAction(MainWindow)
        self.actionOpen_Text_1.setObjectName("actionOpen_Text_1")
        self.actionOpen_Text_2 = QtGui.QAction(MainWindow)
        self.actionOpen_Text_2.setObjectName("actionOpen_Text_2")
        self.actionSave_Text_1 = QtGui.QAction(MainWindow)
        self.actionSave_Text_1.setObjectName("actionSave_Text_1")
        self.actionSave_Text_2 = QtGui.QAction(MainWindow)
        self.actionSave_Text_2.setObjectName("actionSave_Text_2")
        self.actionUndo_in_Text_1 = QtGui.QAction(MainWindow)
        self.actionUndo_in_Text_1.setObjectName("actionUndo_in_Text_1")
        self.actionUndo_in_Text_2 = QtGui.QAction(MainWindow)
        self.actionUndo_in_Text_2.setObjectName("actionUndo_in_Text_2")
        self.actionRedo_in_Text_1 = QtGui.QAction(MainWindow)
        self.actionRedo_in_Text_1.setObjectName("actionRedo_in_Text_1")
        self.actionRedo_in_Text_2 = QtGui.QAction(MainWindow)
        self.actionRedo_in_Text_2.setObjectName("actionRedo_in_Text_2")
        self.actionCut_from_Text_1 = QtGui.QAction(MainWindow)
        self.actionCut_from_Text_1.setObjectName("actionCut_from_Text_1")
        self.actionCut_from_Text_2 = QtGui.QAction(MainWindow)
        self.actionCut_from_Text_2.setObjectName("actionCut_from_Text_2")
        self.actionCopy_from_Text_1 = QtGui.QAction(MainWindow)
        self.actionCopy_from_Text_1.setObjectName("actionCopy_from_Text_1")
        self.actionCopy_from_Text_2 = QtGui.QAction(MainWindow)
        self.actionCopy_from_Text_2.setObjectName("actionCopy_from_Text_2")
        self.actionPaste_into_Text_1 = QtGui.QAction(MainWindow)
        self.actionPaste_into_Text_1.setObjectName("actionPaste_into_Text_1")
        self.actionPaste_into_Text_2 = QtGui.QAction(MainWindow)
        self.actionPaste_into_Text_2.setObjectName("actionPaste_into_Text_2")
        self.actionFind_in_Text_1 = QtGui.QAction(MainWindow)
        self.actionFind_in_Text_1.setObjectName("actionFind_in_Text_1")
        self.actionFind_in_Text_2 = QtGui.QAction(MainWindow)
        self.actionFind_in_Text_2.setObjectName("actionFind_in_Text_2")
        self.actionFont_of_Text_1 = QtGui.QAction(MainWindow)
        self.actionFont_of_Text_1.setObjectName("actionFont_of_Text_1")
        self.actionFont_of_Text_2 = QtGui.QAction(MainWindow)
        self.actionFont_of_Text_2.setObjectName("actionFont_of_Text_2")
        self.actionColor_of_Text_1 = QtGui.QAction(MainWindow)
        self.actionColor_of_Text_1.setObjectName("actionColor_of_Text_1")
        self.actionColor_of_Text_2 = QtGui.QAction(MainWindow)
        self.actionColor_of_Text_2.setObjectName("actionColor_of_Text_2")
        self.actionHighlight_in_Text_1 = QtGui.QAction(MainWindow)
        self.actionHighlight_in_Text_1.setObjectName("actionHighlight_in_Text_1")
        self.actionHighlight_in_Text_2 = QtGui.QAction(MainWindow)
        self.actionHighlight_in_Text_2.setObjectName("actionHighlight_in_Text_2")
        self.actionZoom_In_Text_1 = QtGui.QAction(MainWindow)
        self.actionZoom_In_Text_1.setObjectName("actionZoom_In_Text_1")
        self.actionZoom_In_Text_2 = QtGui.QAction(MainWindow)
        self.actionZoom_In_Text_2.setObjectName("actionZoom_In_Text_2")
        self.actionZoom_Out_Text_1 = QtGui.QAction(MainWindow)
        self.actionZoom_Out_Text_1.setObjectName("actionZoom_Out_Text_1")
        self.actionZoom_Out_Text_2 = QtGui.QAction(MainWindow)
        self.actionZoom_Out_Text_2.setObjectName("actionZoom_Out_Text_2")
        self.actionSelect_All_in_Text_1 = QtGui.QAction(MainWindow)
        self.actionSelect_All_in_Text_1.setObjectName("actionSelect_All_in_Text_1")
        self.actionSelect_All_in_Text_2 = QtGui.QAction(MainWindow)
        self.actionSelect_All_in_Text_2.setObjectName("actionSelect_All_in_Text_2")
        self.actionAbout = QtGui.QAction(MainWindow)
        self.actionAbout.setObjectName("actionAbout")
        self.actionNew_Text_3 = QtGui.QAction(MainWindow)
        self.actionNew_Text_3.setObjectName("actionNew_Text_3")
        self.actionOpen_Text_3 = QtGui.QAction(MainWindow)
        self.actionOpen_Text_3.setObjectName("actionOpen_Text_3")
        self.actionSave_Text_3 = QtGui.QAction(MainWindow)
        self.actionSave_Text_3.setObjectName("actionSave_Text_3")
        self.actionUndo_in_Text_3 = QtGui.QAction(MainWindow)
        self.actionUndo_in_Text_3.setObjectName("actionUndo_in_Text_3")
        self.actionRedo_in_Text_3 = QtGui.QAction(MainWindow)
        self.actionRedo_in_Text_3.setObjectName("actionRedo_in_Text_3")
        self.actionCut_from_Text_3 = QtGui.QAction(MainWindow)
        self.actionCut_from_Text_3.setObjectName("actionCut_from_Text_3")
        self.actionCopy_from_Text_3 = QtGui.QAction(MainWindow)
        self.actionCopy_from_Text_3.setObjectName("actionCopy_from_Text_3")
        self.actionPaste_into_Text_3 = QtGui.QAction(MainWindow)
        self.actionPaste_into_Text_3.setObjectName("actionPaste_into_Text_3")
        self.actionSelect_All_in_Text_3 = QtGui.QAction(MainWindow)
        self.actionSelect_All_in_Text_3.setObjectName("actionSelect_All_in_Text_3")
        self.actionZoom_In_Text_3 = QtGui.QAction(MainWindow)
        self.actionZoom_In_Text_3.setObjectName("actionZoom_In_Text_3")
        self.actionZoom_Out_Text_3 = QtGui.QAction(MainWindow)
        self.actionZoom_Out_Text_3.setObjectName("actionZoom_Out_Text_3")
        self.actionFont_of_Text_3 = QtGui.QAction(MainWindow)
        self.actionFont_of_Text_3.setObjectName("actionFont_of_Text_3")
        self.actionColor_of_Text_3 = QtGui.QAction(MainWindow)
        self.actionColor_of_Text_3.setObjectName("actionColor_of_Text_3")
        self.actionHighlight_in_Text_3 = QtGui.QAction(MainWindow)
        self.actionHighlight_in_Text_3.setObjectName("actionHighlight_in_Text_3")
        self.actionFind_in_Text_3 = QtGui.QAction(MainWindow)
        self.actionFind_in_Text_3.setObjectName("actionFind_in_Text_3")
        self.menuNew.addAction(self.actionNew_Text_1)
        self.menuNew.addAction(self.actionNew_Text_2)
        self.menuNew.addAction(self.actionNew_Text_3)
        self.menuOpen.addAction(self.actionOpen_Text_1)
        self.menuOpen.addAction(self.actionOpen_Text_2)
        self.menuOpen.addAction(self.actionOpen_Text_3)
        self.menuSave.addAction(self.actionSave_Text_1)
        self.menuSave.addAction(self.actionSave_Text_2)
        self.menuSave.addAction(self.actionSave_Text_3)
        self.menuFile.addAction(self.menuNew.menuAction())
        self.menuFile.addSeparator()
        self.menuFile.addAction(self.menuOpen.menuAction())
        self.menuFile.addAction(self.menuSave.menuAction())
        self.menuFile.addSeparator()
        self.menuFile.addAction(self.actionExit)
        self.menuUndo.addAction(self.actionUndo_in_Text_1)
        self.menuUndo.addAction(self.actionUndo_in_Text_2)
        self.menuUndo.addAction(self.actionUndo_in_Text_3)
        self.menuRedo.addAction(self.actionRedo_in_Text_1)
        self.menuRedo.addAction(self.actionRedo_in_Text_2)
        self.menuRedo.addAction(self.actionRedo_in_Text_3)
        self.menuCut.addAction(self.actionCut_from_Text_1)
        self.menuCut.addAction(self.actionCut_from_Text_2)
        self.menuCut.addAction(self.actionCut_from_Text_3)
        self.menuCopy.addAction(self.actionCopy_from_Text_1)
        self.menuCopy.addAction(self.actionCopy_from_Text_2)
        self.menuCopy.addAction(self.actionCopy_from_Text_3)
        self.menuPaste.addAction(self.actionPaste_into_Text_1)
        self.menuPaste.addAction(self.actionPaste_into_Text_2)
        self.menuPaste.addAction(self.actionPaste_into_Text_3)
        self.menuFind.addAction(self.actionFind_in_Text_1)
        self.menuFind.addAction(self.actionFind_in_Text_2)
        self.menuFind.addAction(self.actionFind_in_Text_3)
        self.menuSelect_All.addAction(self.actionSelect_All_in_Text_1)
        self.menuSelect_All.addAction(self.actionSelect_All_in_Text_2)
        self.menuSelect_All.addAction(self.actionSelect_All_in_Text_3)
        self.menuEdit.addAction(self.menuUndo.menuAction())
        self.menuEdit.addAction(self.menuRedo.menuAction())
        self.menuEdit.addSeparator()
        self.menuEdit.addAction(self.menuCut.menuAction())
        self.menuEdit.addAction(self.menuCopy.menuAction())
        self.menuEdit.addAction(self.menuPaste.menuAction())
        self.menuEdit.addAction(self.menuSelect_All.menuAction())
        self.menuEdit.addSeparator()
        self.menuEdit.addAction(self.menuFind.menuAction())
        self.menuFont.addAction(self.actionFont_of_Text_1)
        self.menuFont.addAction(self.actionFont_of_Text_2)
        self.menuFont.addAction(self.actionFont_of_Text_3)
        self.menuColor.addAction(self.actionColor_of_Text_1)
        self.menuColor.addAction(self.actionColor_of_Text_2)
        self.menuColor.addAction(self.actionColor_of_Text_3)
        self.menuHighlight.addAction(self.actionHighlight_in_Text_1)
        self.menuHighlight.addAction(self.actionHighlight_in_Text_2)
        self.menuHighlight.addAction(self.actionHighlight_in_Text_3)
        self.menuTools.addAction(self.menuFont.menuAction())
        self.menuTools.addAction(self.menuColor.menuAction())
        self.menuTools.addAction(self.menuHighlight.menuAction())
        self.menuZoom_In.addAction(self.actionZoom_In_Text_1)
        self.menuZoom_In.addAction(self.actionZoom_In_Text_2)
        self.menuZoom_In.addAction(self.actionZoom_In_Text_3)
        self.menuZoom_Out.addAction(self.actionZoom_Out_Text_1)
        self.menuZoom_Out.addAction(self.actionZoom_Out_Text_2)
        self.menuZoom_Out.addAction(self.actionZoom_Out_Text_3)
        self.menuView.addAction(self.menuZoom_In.menuAction())
        self.menuView.addAction(self.menuZoom_Out.menuAction())
        self.menuHelp.addAction(self.actionAbout)
        self.menubar.addAction(self.menuFile.menuAction())
        self.menubar.addAction(self.menuEdit.menuAction())
        self.menubar.addAction(self.menuView.menuAction())
        self.menubar.addAction(self.menuTools.menuAction())
        self.menubar.addAction(self.menuHelp.menuAction())

        self.retranslateUi(MainWindow)
        QtCore.QMetaObject.connectSlotsByName(MainWindow)

    def retranslateUi(self, MainWindow):
        MainWindow.setWindowTitle(QtGui.QApplication.translate("MainWindow", "ThreeTextEditor", None, QtGui.QApplication.UnicodeUTF8))
        self.label_Text_1.setText(QtGui.QApplication.translate("MainWindow", "<b>Text 1</b>", None, QtGui.QApplication.UnicodeUTF8))
        self.label_Text_2.setText(QtGui.QApplication.translate("MainWindow", "<b>Text 2</b>", None, QtGui.QApplication.UnicodeUTF8))
        self.label_Text_3.setText(QtGui.QApplication.translate("MainWindow", "<b>Text 3</b>", None, QtGui.QApplication.UnicodeUTF8))
        self.menuFile.setTitle(QtGui.QApplication.translate("MainWindow", "File", None, QtGui.QApplication.UnicodeUTF8))
        self.menuNew.setTitle(QtGui.QApplication.translate("MainWindow", "New", None, QtGui.QApplication.UnicodeUTF8))
        self.menuOpen.setTitle(QtGui.QApplication.translate("MainWindow", "Open", None, QtGui.QApplication.UnicodeUTF8))
        self.menuSave.setTitle(QtGui.QApplication.translate("MainWindow", "Save", None, QtGui.QApplication.UnicodeUTF8))
        self.menuEdit.setTitle(QtGui.QApplication.translate("MainWindow", "Edit", None, QtGui.QApplication.UnicodeUTF8))
        self.menuUndo.setTitle(QtGui.QApplication.translate("MainWindow", "Undo", None, QtGui.QApplication.UnicodeUTF8))
        self.menuRedo.setTitle(QtGui.QApplication.translate("MainWindow", "Redo", None, QtGui.QApplication.UnicodeUTF8))
        self.menuCut.setTitle(QtGui.QApplication.translate("MainWindow", "Cut", None, QtGui.QApplication.UnicodeUTF8))
        self.menuCopy.setTitle(QtGui.QApplication.translate("MainWindow", "Copy", None, QtGui.QApplication.UnicodeUTF8))
        self.menuPaste.setTitle(QtGui.QApplication.translate("MainWindow", "Paste", None, QtGui.QApplication.UnicodeUTF8))
        self.menuFind.setTitle(QtGui.QApplication.translate("MainWindow", "Find", None, QtGui.QApplication.UnicodeUTF8))
        self.menuSelect_All.setTitle(QtGui.QApplication.translate("MainWindow", "Select All", None, QtGui.QApplication.UnicodeUTF8))
        self.menuTools.setTitle(QtGui.QApplication.translate("MainWindow", "Tools", None, QtGui.QApplication.UnicodeUTF8))
        self.menuFont.setTitle(QtGui.QApplication.translate("MainWindow", "Font", None, QtGui.QApplication.UnicodeUTF8))
        self.menuColor.setTitle(QtGui.QApplication.translate("MainWindow", "Color", None, QtGui.QApplication.UnicodeUTF8))
        self.menuHighlight.setTitle(QtGui.QApplication.translate("MainWindow", "Highlight", None, QtGui.QApplication.UnicodeUTF8))
        self.menuView.setTitle(QtGui.QApplication.translate("MainWindow", "View", None, QtGui.QApplication.UnicodeUTF8))
        self.menuZoom_In.setTitle(QtGui.QApplication.translate("MainWindow", "Zoom In", None, QtGui.QApplication.UnicodeUTF8))
        self.menuZoom_Out.setTitle(QtGui.QApplication.translate("MainWindow", "Zoom Out", None, QtGui.QApplication.UnicodeUTF8))
        self.menuHelp.setTitle(QtGui.QApplication.translate("MainWindow", "Help", None, QtGui.QApplication.UnicodeUTF8))
        self.actionExit.setText(QtGui.QApplication.translate("MainWindow", "Exit", None, QtGui.QApplication.UnicodeUTF8))
        self.actionNew_Text_1.setText(QtGui.QApplication.translate("MainWindow", "New Text 1", None, QtGui.QApplication.UnicodeUTF8))
        self.actionNew_Text_2.setText(QtGui.QApplication.translate("MainWindow", "New Text 2", None, QtGui.QApplication.UnicodeUTF8))
        self.actionOpen_Text_1.setText(QtGui.QApplication.translate("MainWindow", "Open Text 1", None, QtGui.QApplication.UnicodeUTF8))
        self.actionOpen_Text_2.setText(QtGui.QApplication.translate("MainWindow", "Open Text 2", None, QtGui.QApplication.UnicodeUTF8))
        self.actionSave_Text_1.setText(QtGui.QApplication.translate("MainWindow", "Save Text 1", None, QtGui.QApplication.UnicodeUTF8))
        self.actionSave_Text_2.setText(QtGui.QApplication.translate("MainWindow", "Save Text 2", None, QtGui.QApplication.UnicodeUTF8))
        self.actionUndo_in_Text_1.setText(QtGui.QApplication.translate("MainWindow", "Undo in Text 1", None, QtGui.QApplication.UnicodeUTF8))
        self.actionUndo_in_Text_2.setText(QtGui.QApplication.translate("MainWindow", "Undo in Text 2", None, QtGui.QApplication.UnicodeUTF8))
        self.actionRedo_in_Text_1.setText(QtGui.QApplication.translate("MainWindow", "Redo in Text 1", None, QtGui.QApplication.UnicodeUTF8))
        self.actionRedo_in_Text_2.setText(QtGui.QApplication.translate("MainWindow", "Redo in Text 2", None, QtGui.QApplication.UnicodeUTF8))
        self.actionCut_from_Text_1.setText(QtGui.QApplication.translate("MainWindow", "Cut from Text 1", None, QtGui.QApplication.UnicodeUTF8))
        self.actionCut_from_Text_2.setText(QtGui.QApplication.translate("MainWindow", "Cut from Text 2", None, QtGui.QApplication.UnicodeUTF8))
        self.actionCopy_from_Text_1.setText(QtGui.QApplication.translate("MainWindow", "Copy from Text 1", None, QtGui.QApplication.UnicodeUTF8))
        self.actionCopy_from_Text_2.setText(QtGui.QApplication.translate("MainWindow", "Copy from Text 2", None, QtGui.QApplication.UnicodeUTF8))
        self.actionPaste_into_Text_1.setText(QtGui.QApplication.translate("MainWindow", "Paste into Text 1", None, QtGui.QApplication.UnicodeUTF8))
        self.actionPaste_into_Text_2.setText(QtGui.QApplication.translate("MainWindow", "Paste into Text 2", None, QtGui.QApplication.UnicodeUTF8))
        self.actionFind_in_Text_1.setText(QtGui.QApplication.translate("MainWindow", "Find in Text 1", None, QtGui.QApplication.UnicodeUTF8))
        self.actionFind_in_Text_2.setText(QtGui.QApplication.translate("MainWindow", "Find in Text 2", None, QtGui.QApplication.UnicodeUTF8))
        self.actionFont_of_Text_1.setText(QtGui.QApplication.translate("MainWindow", "Font of Text 1", None, QtGui.QApplication.UnicodeUTF8))
        self.actionFont_of_Text_2.setText(QtGui.QApplication.translate("MainWindow", "Font of Text 2", None, QtGui.QApplication.UnicodeUTF8))
        self.actionColor_of_Text_1.setText(QtGui.QApplication.translate("MainWindow", "Color of Text 1", None, QtGui.QApplication.UnicodeUTF8))
        self.actionColor_of_Text_2.setText(QtGui.QApplication.translate("MainWindow", "Color of Text 2", None, QtGui.QApplication.UnicodeUTF8))
        self.actionHighlight_in_Text_1.setText(QtGui.QApplication.translate("MainWindow", "Highlight in Text 1", None, QtGui.QApplication.UnicodeUTF8))
        self.actionHighlight_in_Text_2.setText(QtGui.QApplication.translate("MainWindow", "Highlight in Text 2", None, QtGui.QApplication.UnicodeUTF8))
        self.actionZoom_In_Text_1.setText(QtGui.QApplication.translate("MainWindow", "Zoom In Text 1", None, QtGui.QApplication.UnicodeUTF8))
        self.actionZoom_In_Text_2.setText(QtGui.QApplication.translate("MainWindow", "Zoom In Text 2", None, QtGui.QApplication.UnicodeUTF8))
        self.actionZoom_Out_Text_1.setText(QtGui.QApplication.translate("MainWindow", "Zoom Out Text 1", None, QtGui.QApplication.UnicodeUTF8))
        self.actionZoom_Out_Text_2.setText(QtGui.QApplication.translate("MainWindow", "Zoom Out Text 2", None, QtGui.QApplication.UnicodeUTF8))
        self.actionSelect_All_in_Text_1.setText(QtGui.QApplication.translate("MainWindow", "Select All in Text 1", None, QtGui.QApplication.UnicodeUTF8))
        self.actionSelect_All_in_Text_2.setText(QtGui.QApplication.translate("MainWindow", "Select All in Text 2", None, QtGui.QApplication.UnicodeUTF8))
        self.actionAbout.setText(QtGui.QApplication.translate("MainWindow", "About", None, QtGui.QApplication.UnicodeUTF8))
        self.actionNew_Text_3.setText(QtGui.QApplication.translate("MainWindow", "New Text 3", None, QtGui.QApplication.UnicodeUTF8))
        self.actionOpen_Text_3.setText(QtGui.QApplication.translate("MainWindow", "Open Text 3", None, QtGui.QApplication.UnicodeUTF8))
        self.actionSave_Text_3.setText(QtGui.QApplication.translate("MainWindow", "Save Text 3", None, QtGui.QApplication.UnicodeUTF8))
        self.actionUndo_in_Text_3.setText(QtGui.QApplication.translate("MainWindow", "Undo in Text 3", None, QtGui.QApplication.UnicodeUTF8))
        self.actionRedo_in_Text_3.setText(QtGui.QApplication.translate("MainWindow", "Redo in Text 3", None, QtGui.QApplication.UnicodeUTF8))
        self.actionCut_from_Text_3.setText(QtGui.QApplication.translate("MainWindow", "Cut from Text 3", None, QtGui.QApplication.UnicodeUTF8))
        self.actionCopy_from_Text_3.setText(QtGui.QApplication.translate("MainWindow", "Copy from Text 3", None, QtGui.QApplication.UnicodeUTF8))
        self.actionPaste_into_Text_3.setText(QtGui.QApplication.translate("MainWindow", "Paste into Text 3", None, QtGui.QApplication.UnicodeUTF8))
        self.actionSelect_All_in_Text_3.setText(QtGui.QApplication.translate("MainWindow", "Select All in Text 3", None, QtGui.QApplication.UnicodeUTF8))
        self.actionZoom_In_Text_3.setText(QtGui.QApplication.translate("MainWindow", "Zoom In Text 3", None, QtGui.QApplication.UnicodeUTF8))
        self.actionZoom_Out_Text_3.setText(QtGui.QApplication.translate("MainWindow", "Zoom Out Text 3", None, QtGui.QApplication.UnicodeUTF8))
        self.actionFont_of_Text_3.setText(QtGui.QApplication.translate("MainWindow", "Font of Text 3", None, QtGui.QApplication.UnicodeUTF8))
        self.actionColor_of_Text_3.setText(QtGui.QApplication.translate("MainWindow", "Color of Text 3", None, QtGui.QApplication.UnicodeUTF8))
        self.actionHighlight_in_Text_3.setText(QtGui.QApplication.translate("MainWindow", "Highlight in Text 3", None, QtGui.QApplication.UnicodeUTF8))
        self.actionFind_in_Text_3.setText(QtGui.QApplication.translate("MainWindow", "Find in Text 3", None, QtGui.QApplication.UnicodeUTF8))

